finality-aleph/src/new_network/bare.rs

```diff
@@ -0,0 +1,42 @@
+use crate::new_network::{DataNetwork, SendError};
+use aleph_bft::Recipient;
+use codec::Codec;
+use futures::channel::mpsc;
+use std::sync::Arc;
+use tokio::sync::Mutex;
+
+/// For sending arbitrary messages.
+pub trait Sender<D: Clone + Codec + Send>: Clone {
```

Comment:
Why pub? I assume there is reason why not private, but don't get why it can't be `pub(crate)`?

Reply:
Also there are plenty of times you are using `pub`, so please think about this comment as to all of them

Reply:
I keep all of this in non-`pub` modules, so it shouldn't be accessible from outside the crate anyway. And this better illustrates which things are parts of the interface of the module, and avoids the ugly `(crate)`.

I think `pub(crate)` should only be used in a module that you want to be public when a specific item shouldn't, but in general this seems like the design failed at some point already.